src/commands/InitCommand.js

```diff
@@ -32,23 +32,17 @@ export default class InitCommand extends Command {
     }
 
     let targetDependencies;
-    if (packageJson.dependencies && packageJson.dependencies.lerna) {
-      // lerna is a dependency in the current project
-      targetDependencies = packageJson.dependencies;
+    if (packageJson.dependencies && packageJson.dependencies.asini) {
+      // asini is a dependency in the current project
+      targetDependencies = "dependencies";
     } else {
-      // lerna is a devDependency or no dependency, yet
+      // asini is a devDependency or no dependency, yet
       if (!packageJson.devDependencies) packageJson.devDependencies = {};
-      targetDependencies = packageJson.devDependencies;
+      targetDependencies = "devDependencies";
     }
 
-    objectAssignSorted(targetDependencies, {
-      lerna: this.lernaVersion
-    });
```

Comment:
Oh, man... what is this I don't even... :squint:

I must have botched a port of a Lerna patch somehow.

Thanks for cleaning this up while you were in here @evocateur!